notebooks/3.jump-analysis/nbconverted/3.jump_analysis.py

```diff
@@ -361,23 +365,65 @@
     lambda injury_code: injury_decoder[str(injury_code)]
 )
 
-# next only select cytoskeletal probability scores
-cytoskeletal_proba_scores = all_proba_scores[col_to_sel + ["Cytoskeletal"]]
-cytoskeletal_proba_scores.rename(columns={"Cytoskeletal": "Cytoskeletal Proba"})
 
+# We will now save the ground truth predictions, which include probability scores for each injury type and model type, as well as the predicted injury. These results will be stored in the `./results/3.jump_analysis` directory.
 
 # In[14]:
 
 
-# Saving only cyr
+# update columns by replacing the column index (which are the injury codes) to the injury type
+gt_y_proba_df.columns = [
+    injury_decoder[str(injury_code)]
+    for injury_code in gt_y_proba_df.columns.tolist()[0:15]
```

Comment:
Recommend not hard-coding index values if possible e.g. [0:15]

Reply:
Thanks for the suggestion!

The hard-coded values for detecting the 15 cellular injury types will remain unchanged because these are fixed and specific to our dataset. An alternative approach, such as creating a separate list to store the injury type names, would introduce redundancy without adding significant flexibility or maintainability to the code. Therefore, the current hard-coding approach is both efficient and appropriate for our use case.